webui/src/app/stores/applications-store.service.ts

```diff
@@ -4,7 +4,8 @@ import { createFeatureSelector, createSelector, Selector, Store } from '@ngrx/st
 import { ActionWithPayloadAndPromise } from './action-with-payload';
 import { ApplicationBean, ApplicationPagesBean,
   DeploymentBean, DomainBean, DomainPagesBean, BadgeRatingBean } from '../models/commons/applications-bean';
-import { Subject } from 'rxjs/Subject';
+import { Subject } from 'rxjs';
+import { Observable } from 'rxjs/internal/Observable';
```

Comment:
Sounds very weird to rely on someone else' internals! Auto-import typo for `rxjs/Observable`?